chpl/chpl-service/src/test/java/gov/healthit/chpl/manager/impl/CertifiedProductManagerTest.java

```diff
@@ -2192,16 +2192,16 @@ public void testAddTestTask()
         TestTask toAdd = new TestTask();
         toAdd.setId(-700L);
         toAdd.setDescription("description of the task");
-        toAdd.setTaskErrors(6.5F);
-        toAdd.setTaskErrorsStddev(4.5F);
+        toAdd.setTaskErrors("6.5");
+        toAdd.setTaskErrorsStddev("4.5");
         toAdd.setTaskPathDeviationObserved("123");
         toAdd.setTaskPathDeviationOptimal("0");
-        toAdd.setTaskRating(5.0F);
+        toAdd.setTaskRating("5.0");
         toAdd.setTaskRatingScale("Likert");
-        toAdd.setTaskRatingStddev(1.5F);
-        toAdd.setTaskSuccessAverage(95.1F);
-        toAdd.setTaskSuccessStddev(90.0F);
-        toAdd.setTaskTimeAvg("1.5");
+        toAdd.setTaskRatingStddev("1.5");
+        toAdd.setTaskSuccessAverage("95.1");
+        toAdd.setTaskSuccessStddev("90.0");
+        toAdd.setTaskTimeAvg("15");
```

Comment:
Could you add a unit test for the reviewer that makes one or more of these fields not a valid number and checks for the new error messages?

Reply:
Added a single test. It's super basic, and there are a lot of errors in the mock listing, but it's a start